public/static/js/spawned_hosts.js

```diff
@@ -721,6 +723,7 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q',
       $location.search("id", host.id);
       $scope.curHostData = host;
       $scope.curHostData.isTerminated = host.isTerminated;
+      $scope.curHostData.isStarted = host.isStarted;
```

Comment:
why is this (and the line before) necessary?

Reply:
oh interesting. I think they are not haha